Hestia/app/src/main/java/hestia/backend/exceptions/ServerExceptions/ServerException.java

```diff
@@ -0,0 +1,14 @@
+package hestia.backend.exceptions.ServerExceptions;
+
+public abstract class ServerException extends Exception {
+    String error;
+
+    public ServerException(String message){
+        this.error=message;
+    }
```

Comment:
Exceptions comming from the server no longer always have a message field. This was discussed last sunday.